azure-pipelines/build.yml

```diff
@@ -22,6 +23,57 @@ jobs:
   - ${{ if eq(variables['system.collectionId'], '011b8bdf-6d56-4f87-be0d-0092136884d9') }}:
     # https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/25351/APIScan-step-by-step-guide-to-setting-up-a-Pipeline
     - group: VSCloudServices-APIScan # Expected to provide ApiScanClientId, ApiScanSecret, ApiScanTenant
+  templateContext:
+    mb:
+      signing:
+        enabled: true
+        signType: $(SignType)
+        zipSources: false
+      sbom:
+        enabled: true
```

Comment:
If this replaces the traditional MicroBuild tasks, we should remove them from the `microbuild.before.yml` file.

Reply:
I thought I had.  Thanks.